internal/website/content/howto/runtimervar/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Runtime Configuration"
+date: 2019-06-18T14:51:12-07:00
+showInSidenav: true
+---
+
+Working with runtime variables using the Go CDK takes two steps:
+
+1. Open the variable with the `runtimevar` provider of your choice. You can
+   provide a [decoder][] parameter to specify how the value stored in the
+   variable is interpreted.
+2. Use the `Latest` method to fetch the value of the variable.
```

Comment:
Consider "value" -> "interpreted value"?

Reply:
I prefer the existing text to keep it simple. The previous sentence already describes "interpreted" so no reason to repeat it here, IMO